### PR TITLE
feat(textures): worker for ktx2 basis writer

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

**Review by ibgreen — APPROVED** [review]

File: modules/textures/package.json [hunk 1]
```diff
@@ -48,5 +50,8 @@
     "@loaders.gl/worker-utils": "3.2.0-alpha.1",
     "ktx-parse": "^0.0.4",
     "texture-compressor": "^1.0.2"
+  },
+  "devDependencies": {
+    "@loaders.gl/polyfills": "3.2.0-alpha.1"
```
[the remark this answers is not in the document]
Comment (belom88) [reply modules/textures/package.json:55]: Done in dedicated PR

File: modules/textures/package.json [hunk 2]
```diff
@@ -35,11 +35,13 @@
...
     "build-compressed-texture-worker": "esbuild src/workers/compressed-texture-worker.ts --bundle --outfile=dist/compressed-texture-worker.js --define:__VERSION__=\\\"$npm_package_version\\\"",
-    "build-crunch-worker": "esbuild src/workers/crunch-worker.ts --bundle --outfile=dist/crunch-worker.js --define:__VERSION__=\\\"$npm_package_version\\\""
+    "build-crunch-worker": "esbuild src/workers/crunch-worker.ts --bundle --outfile=dist/crunch-worker.js --define:__VERSION__=\\\"$npm_package_version\\\"",
+    "build-ktx2-basis-universal-texture-writer-worker": "esbuild src/workers/ktx2-basis-universal-texture-writer-worker.ts --bundle --outfile=dist/ktx2-basis-universal-texture-writer-worker.js --define:__VERSION__=\\\"$npm_package_version\\\"",
+    "build-ktx2-basis-universal-texture-writer-nodejs-worker": "esbuild src/workers/ktx2-basis-universal-texture-writer-nodejs-worker.ts --platform=node --target=esnext,node12 --bundle --outfile=dist/ktx2-basis-universal-texture-writer-nodejs-worker.js --define:__VERSION__=\\\"$npm_package_version\\\""
```
[the remark this answers is not in the document]
Comment (belom88) [reply modules/textures/package.json:44]: `-worker` should be in the end of the name

File: modules/textures/src/ktx2-basis-universal-texture-writer.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import {encodeKTX2BasisTexture} from './lib/encoders/encode-ktx2-basis-texture';
  */
 export const KTX2BasisUniversalTextureWriter = {
   name: 'Basis Universal Supercompressed GPU Texture',
-  id: 'ktx2-basis-supercompressed-texture',
+  id: 'ktx2-basis-universal-texture-writer',
```
[the remark this answers is not in the document]
Comment (belom88) [reply modules/textures/src/ktx2-basis-universal-texture-writer.ts:11]: done

File: modules/textures/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (belom88) [reply modules/textures/package.json]: https://github.com/visgl/loaders.gl/blob/de03da208acfa5216e2e2a7d91dccd8147cd0be8/modules/worker-utils/src/lib/worker-api/get-worker-url.ts#L26

**Pull request merged**